certified-connectors/Datablend/readme.MD

```diff
@@ -0,0 +1,17 @@
+## Datablend API Connector
+Datablend is a powerful iPaaS solution with a focus is serving the needs of CFOs, controllers, and financial planning and analysis (budgeting) professionals as they look to streamline processes, speed up closes, and increase confidence in reporting.
+
```

Comment:
@bmp02050 Please refer the templates and add required sections, for example Publisher section is missing.

Reply:
This is completed

Reply:
@bmp02050 The publisher name section is missing. , [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) is the template you can refer to update your connector's readme.md file.